tests/unit/querybuf.tcl

```diff
@@ -1,32 +1,39 @@
 start_server {tags {"querybuf"}} {
-    test "querybuf will never be resized when querybuf size less than 64k" {
+    test "query buffer will never be resized when less than 64k" {
         set orig_used [s used_memory]
 
-        # Memory will increase by more than 32k due to querybuf of new client.
+        # Memory will increase by more than 32k due to query buffer of client.
         set rd [redis_deferring_client]
         assert_morethan [expr [s used_memory] - $orig_used] 32768
 
-        after 1000
+        if {$::accurate} {
+            after 2000 ;# Check if client query buffer will be resized when client idle 2s
+        } else {
+            after 120 ;# serverCron only call clientsCronResizeQueryBuffer once in 100ms
+        }
 
-        # Check querybuf of new client has not been resized.
+        # Check query buffer of client has not been resized.
         assert_morethan [expr [s used_memory] - $orig_used] 32768
     }
 
-    test "querybuf will be resized when querybuf size more than 64k" {
-        set rd1 [redis_deferring_client]
-        $rd1 set bigstring [string repeat A 65535] ;# Create a 64k bigstring in advance.
-        $rd1 close
+    test "query buffer will be resized when more than 64k" {
+        # The test will run at least 2s to wait for client query
+        # buffer to be resized after idle 2s.
+        if {$::accurate} {
```

Comment:
i know @yossigo is cooking some improvements to the `--tags` system, and mentioned that maybe a better approach would be to use the `slow` tag.
the disadvantage in my eyes is that it'll be opt-out rather than opt-in.
but i guess that's already the case with other slow test, like the ones in sort.tcl, expire.tcl etc (that are already tagged).

p.s. if we keep that `if` it should be outside the `test` declaration (not inside it)

Reply:
I guess ```-tag`` is used to execute a certain series of tags, which should not be applicable to slow tests.